janus_core/cli/geomopt.py

```diff
@@ -113,17 +113,23 @@ def geomopt(
             rich_help_panel="Calculation",
         ),
     ] = False,
-    filter_func: Annotated[
+    filter_class: Annotated[
         str | None,
         Option(
             help=(
-                "Name of ASE filter/constraint function to use. If using "
+                "Name of ASE filter to wrap around atoms. If using "
                 "--opt-cell-lengths or --opt-cell-fully, defaults to "
                 "`FrechetCellFilter`."
             ),
             rich_help_panel="Calculation",
         ),
     ] = None,
+    filter_func: Annotated[
```

Comment:
https://typer.tiangolo.com/tutorial/commands/help/#deprecate-a-command ?

Reply:
I did have a look at this, but I think it only applies to CLI commands, rather than their parameters. I think click has introduced a parameter deprecated option as of 8.2.0 (https://github.com/pallets/click/pull/2271), so when typer is updated for this, that may be useful in the future.

Reply:
I'll open an issue, but given that we don't plan to keep these as deprecated for very long, and it's unclear how long it'll take for typer to become compatible, I'm not sure it'll be worth it for these cases.